geomap/src/components/LeafletMap.vue

```diff
@@ -56,24 +59,67 @@ import {
 import MarkerCluster from "./MarkerCluster.vue";
 import { Vue } from "vue-class-component";
 import { useStore } from "vuex";
+import L from "leaflet";
```

Comment:
`vue-leaflet` is using the esm version isn't it?  Shouldn't this just `import { Icon } from "leaflet/dist/leaflet-src.esm"` like the other esm-using code does?

Reply:
Resolved